heal-remedy.lic

```diff
@@ -0,0 +1,93 @@
+# quiet
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#tendme
```

Comment:
`tendme` should be `heal-remedy`

Reply:
Fixed.

Reply:
I was also thinking, maybe with a yaml setting, one could call this from within tendme. Thoughts?